src/main/scala/net/ripe/rpki/nro/service/Ports.scala

```diff
@@ -212,9 +230,30 @@ object Ports extends Logging {
     }
   }
 
-  def readConflicts(conflictFile: String = s"${config.previousConflictFile}"): List[Conflict] = {
-    logger.debug(s"Reading conflicts from ${config.previousConflictFile}")
-    Using.resource(CSVReader.open(new File(conflictFile))){ reader =>
+  private def fetchAndWriteLocally(source: String, dest: String): Unit = {
+    val fetchResponse: Future[Response] = fetchWithRetries(source)
+    Try(Await.result(fetchResponse, Duration.Inf)) match {
```

Comment:
It's best to set a timeout and not wait indefinitely

Reply:
Waited for 2  mins.